src/Components/Samples/BlazorUnitedApp/Program.cs

```diff
@@ -27,6 +27,7 @@
 app.UseStaticFiles();
 app.UseAntiforgery();
 
+app.MapStaticAssets();
```

Comment:
Yes, they co-exist and the order determines which one applies. If `UseStaticFiles` is called before `UseRouting` matching paths will use that, otherwise, if the file was part of the manifest it will match that and fallback to `UseStaticFiles` if not.